recipes.js

```diff
@@ -1,10 +1,94 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js');
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const data = require("./data.js");
+const Recipe = require("./models/recipe-model.js");
 
-mongoose.connect('mongodb://localhost/recipeApp')
+const terminate = mongoose.connection.close();
+
+mongoose
+  .connect("mongodb://localhost/recipeApp")
   .then(() => {
-    console.log('Connected to Mongo!');
-  }).catch(err => {
-    console.error('Error connecting to mongo', err);
+    console.log("Connected to Mongo!");
+  })
+  .catch(err => {
+    console.error("Error connecting to mongo", err);
   });
+
+function creation() {
+  Recipe.create({
+    title: "Cocktail toasts",
+    level: "Easy Peasy",
+    ingredients: [
+      "shrimps",
+      "avocado",
+      "sliced bread",
+      "orange juice",
+      "whisky",
+      "garlic",
+      "onion",
+      "pepper",
+      "butter"
+    ],
+    cuisine: "Fusion food",
+    dishType: "Other",
+    image: "https://images.media-allrecipes.com/images/75131.jpg",
+    duration: 40,
+    creator: "Bastien Tran",
+    created: new Date()
+  })
+
+    // then() callbacks get called if the operation succeeded
+    .then(recipeDoc => {
+      console.log("Recipe CREATE success!", recipeDoc);
+      batchInsert();
+    })
+
+    // catch() callbacks get called if the operation FAILS
+    .catch(err => {
+      console.log("Recipe CREATE failure!", err);
+    });
+}
+function batchInsert() {
+  Recipe.insertMany(data)
+    .then(recipeDoc => {
+      console.log("Recipe BATCH INSERT success!", recipeDoc.__title);
+      batchUpdate();
+    })
+    .catch(err => {
+      console.log("Recipe BATCH INSERT failure!", err);
+    });
+}
```

Comment:
You did the exercise in a more complex way. But that's totally ok! That was what we were waiting for (very simple)

```
Recipe.insertMany(data)
  .then(recipeDoc => {
    console.log("All the recipes from data.js have been pushed 🥗", recipeDoc);
  })
  .catch(err => {
    console.log("Push the data.js file Failed! 🍎🍐", err);
  });
```